apps/files/src/components/FileDetails.vue

```diff
@@ -85,7 +85,11 @@ export default {
     },
 
     currentTab() {
-      return this.currentSidebarTab?.tab || this.defaultTab
+      if (this.currentTab) {
```

Comment:
Here and in the next line it has to be `this.currentSidebarTab` instead of `this.currentTab`